src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,315 @@
+# Feedback API
+
+Host: https://sentry.io/api/0
+
+**Authors.**
+
+@cmanallen
+
+**How to read this document.**
+
+This document is structured by resource with each resource having actions that
+can be performed against it. Every action that either accepts a request or
+returns a response _will_ document the full interchange format. Clients may opt
+to restrict response data or provide a subset of the request data. The API may
+or may not accept partial payloads.
+
+## Feedback Index [/organizations/<organization_slug>/feedback/]
+
+- Parameters
+
+  - field (optional, string)
+  - environment (optional, string)
+  - project (optional, string)
+  - sort (optional, string) Default: -timestamp Members
+    - projectId
+    - -projectId
+    - timestamp
+    - -timestamp
+  - statsPeriod (optional, string) - A positive integer suffixed with a unit
+    type. Default: 7d Members
+    - s
+    - m
+    - h
+    - d
+    - w
+  - start (optional, string) - ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)
+  - end (optional, string) - ISO 8601 format. Required if `start` is set.
+  - utc (optiona, 'true') - Whether start/end should use the UTC timezone
+  - cursor (optional, str)
```

Comment:
we might need to convert our cursor from being a offset thing to a greater-than/less-than timestamp based one.

Reply:
Let's document the requirement and get buy in before we add this.

Reply:
relates to the infinite scrolling figma design: https://www.figma.com/file/ga57zyLonVCSJIH3kOWLVA/Specs%3A-Bug-Reports?type=design&node-id=3-13295&mode=design&t=9FxkBVPALn6lqTuP-0